### PR TITLE
feat(arch): arm64 image

### LINKED ISSUE #205: any planned support for Apple M1 Chip  linux/arm64
I tried to run up the docker image but it just froze on my Mac with M1 chip when trying to access http://localhost:8080/camunda-welcome/index.html. I noticed you dont build  arm64 distro so was wondering if this was planned and assuming this is the reason for the freezing.

```
docker run -d --name camunda -p 8080:8080 camunda/camunda-bpm-platform:latest

WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

```
```
 docker pull camunda/camunda-bpm-platform:latest --platform linux/arm64       
latest: Pulling from camunda/camunda-bpm-platform
Digest: sha256:81a8a997cd904232d3faf3d14db38d16d43cfc6f05a894a61ac3235eb6c2acbd
Status: Image is up to date for camunda/camunda-bpm-platform:latest
WARNING: image with reference camunda/camunda-bpm-platform was found but does not match the specified platform: wanted linux/arm64, actual: linux/amd64
docker.io/camunda/camunda-bpm-platform:latest

```

### PR DESCRIPTION
**Pull request by megglos**
WIP not ready yet.

Would resolve #205 
Relates to #CAM-14301

See
https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images
https://docs.docker.com/ci-cd/github-actions/
https://github.com/docker/setup-buildx-action#with-qemu

### CODE REVIEW

File: .github/workflows/build-test-and-publish-ce.yml [hunk 1]
```diff
@@ -6,13 +6,25 @@ jobs:
...
+      - name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: arm64,amd64
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        id: buildx
+        with:
+          install: true
```
Comment (megglos) [inline .github/workflows/build-test-and-publish-ce.yml:27]: this just ensured the final images pushed are built multiarch so the test pipeline is actually not making use of those.

This is a bit more tricky to achieve as we would need to also use qemu to test non amd64 images, something like https://github.com/uraimo/run-on-arch-action might be helpful, but this one doesn't support the amd64 arch yet but they are working on it: https://github.com/uraimo/run-on-arch-action/pull/58

File: .github/workflows/build-test-and-publish-ce.yml [hunk 2]
```diff
@@ -7,17 +7,27 @@ jobs:
...
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        id: buildx
       - name: Publish image
         if: ${{ github.ref == 'refs/heads/next' || startsWith(github.ref, 'refs/heads/7.') }}
         run: ./release.sh
         env:
           DISTRO: ${{ matrix.DISTRO }}
           EE: false
+          PLATFORMS: linux/arm64,linux/amd64
```
Comment (megglos) [inline .github/workflows/build-test-and-publish-ce.yml:31]: this just achieves making the final images pushed multiarch so the test pipeline is actually not making use of those, leaving the chance for undetected broken arm64 images. 

Testing of each arch is a bit more tricky to achieve as we would need to also use qemu to test non amd64 images, something like https://github.com/uraimo/run-on-arch-action might be helpful, but this one doesn't support the amd64 arch yet, however they are working on it: https://github.com/uraimo/run-on-arch-action/pull/58

**Comment by ghost** [comment]
When will this be merged? Would be great to have the arm image for the Apple Silicon.

**Comment by patrick-werner** [comment]
Waiting for this to get merged, the x86 image isn't usable on Apple Silicon

**Comment by megglos** [comment]
closing as this will be done by the platform team

**Pull request closed without merging**